src/opencl/mod.rs

```diff
@@ -94,8 +63,7 @@ pub struct Device {
     name: String,
     memory: u64,
     bus_id: Option<BusId>,
-    platform: ocl::Platform,
-    pub device: ocl::Device,
+    pub device: opencl3::device::Device,
```

Comment:
can you add a todo, to make this private? it is very odd to have a single field be public